card/mod/basic_formats/set/self/head.rb

```diff
@@ -34,7 +34,7 @@ def head_buttons
     if root.card
       bits << universal_edit_button
       # RSS # move to mods!
-      bits << rss_link if root.card.type_id == SearchTypeID
+      bits << rss_link
```

Comment:
rss_link_tag moved to search type (imo, these type_id `if` tests are often a red flag)